lib/ui/pointer.dart

```diff
@@ -177,7 +177,8 @@ class PointerData {
     this.panDeltaY = 0.0,
     this.scale = 0.0,
     this.rotation = 0.0,
-  });
+    void Function({bool allowPlatformDefault})? respond,
```

Comment:
This and _respond should probably follow our naming rules: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#naming-rules-for-typedefs-and-function-variables

I think this should be named `onRespond` or something similar.

Reply:
Ah yes, I'll change this and give the callback a better name. The public API on the PointerData will still be called "respond" but the callback we pass in will be called "onRespond"